### PR TITLE
Add Restaurant Component and Firebase retrieval of data

### PR DESCRIPTION
**Pull request by DonkeyKongJr**
Add Restaurant Component and Menu Navigation Entry.
Add Route to Router for restaurants.
Add Retrieval of All Restaurants stored in Firebase via state management.
Add FontAwesome Icons and replacing all existing menu items with FA.
Add Location to Restaurant and Table View.

### CODE REVIEW

File: src/pimlify-vue/src/main.ts [hunk 1]
```diff
@@ -10,17 +10,35 @@ import 'firebase/firestore';
...
+Vue.component('font-awesome-icon', FontAwesomeIcon);
+
 Vue.use(Vuetify);
 Vue.use(VueFire);
 Vue.use(VueRouter);
 
 firebase.initializeApp(firebaseConfig);
 
 export const db = firebase.firestore();
+const settings = { timestampsInSnapshots: true };
```
Comment (DonkeyKongJr) [inline src/pimlify-vue/src/main.ts]: Recommended from Firebase via browser console output.

**Comment by DonkeyKongJr** [comment]
Should complete issue #8. 

For every further feature which uses firebase a additional issue shall be created.

File: src/pimlify-vue/src/components/RestaurantComponent.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+</v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+
+@Component
+export default class RestaurantComponent extends Vue {
+  data() {
```
Comment (Opiskull) [inline src/pimlify-vue/src/components/RestaurantComponent.vue]: Could simply be public headers without data

File: src/pimlify-vue/src/store/actions.ts [hunk 3]
```diff
@@ -1,29 +1,47 @@
...
-      commit("setCurrentOrderMenu", { item: menus[0] });
+      commit('setCurrentOrderMenu', { item: menus[0] });
     }
+  },
+  loadAllRestaurantsFromFireBase({ commit, state }) {
+    let restaurants: Restaurant[] = [];
+
+    db.collection('restaurant')
+      .get()
+      .then(function(querySnapshot) {
```
Comment (Opiskull) [inline src/pimlify-vue/src/store/actions.ts]: Maybe replace the function with arrow

File: src/pimlify-vue/src/components/RestaurantComponent.vue (hunk 2 above)
Comment (DonkeyKongJr) [reply src/pimlify-vue/src/components/RestaurantComponent.vue]: Isn't it best practice in vue components to use data?

File: src/pimlify-vue/src/store/actions.ts (hunk 3 above)
Comment (DonkeyKongJr) [reply src/pimlify-vue/src/store/actions.ts]: Agree

**Pull request merged**